src/components/content/order/Navigate.tsx

```diff
@@ -5,17 +5,18 @@
 
 import { useNavigate, To } from 'react-router-dom';
 import '../../../styles/service_order.css';
+import { OrderSubmitProps } from './OrderSubmit';
 
-function Navigate({ text, to }: { text: string; to: To }): JSX.Element {
+function Navigate({ text, to, serviceInfo }: { text: string; to: To; serviceInfo?: OrderSubmitProps }): JSX.Element {
     const navigate = useNavigate();
 
-    function goBack() {
-        navigate(to);
+    function goBack(serviceInfo: OrderSubmitProps | undefined) {
+        navigate(to, { state: { serviceInfo: serviceInfo } });

```

Comment:
```suggestion
        navigate(to, { state:  serviceInfo });
```
How about making it like this?

Reply:
fixed.